packages/integrations/vercel/README.md

```diff
@@ -85,13 +85,14 @@ vercel deploy --prebuilt
 
 To configure this adapter, pass an object to the `vercel()` function call in `astro.config.mjs`:
 
-### analytics
+### Web Analytics
 
-**Type:** `boolean`<br>
-**Available for:** Serverless, Static<br>
-**Added in:** `@astrojs/vercel@3.1.0`
+**Type:** `VercelWebAnalyticsConfig`<br>
+**Available for:** Serverless, Edge, Static<br>
+**Added in:** `@astrojs/vercel@3.8.0`
 
-You can enable [Vercel Analytics](https://vercel.com/analytics) (including Web Vitals and Audiences) by setting `analytics: true`. This will inject Vercel’s tracking scripts into all your pages.
+You can enable [Vercel Web Analytics](https://vercel.com/docs/concepts/analytics) by setting `webAnalytics: { enabled: true }`. This will inject Vercel’s tracking scripts into all of your pages.
+Alternatively, you can also pass [configuration options](https://vercel.com/docs/concepts/analytics/package) (except functions like `beforeSend`) via the `config` property inside `webAnalytics`.
```

Comment:
```suggestion
You can also pass [Vercel's analytics configuration options](https://vercel.com/docs/concepts/analytics/package) `mode` and `debug` via the `config` property inside `webAnalytics`.
```
- Checking on "alternatively" because I wouldn't think that passing an additional config option is an *alternative* to enabling `webAnalytics`? Should this perhaps be "Additionally" (or, nothing at all, since there's already an "also"?)
- Following that link, there really are only three config options mentioned, so assuming that's all there is (?) it I think we can just state the two which *can* be passed here. Then, there is a section below for how to handle `beforeSend`.

Reply:
Thanks for the callout, you're right. I've updated the copy!